jina/serve/runtimes/gateway/request_handling.py

```diff
@@ -185,20 +192,12 @@ async def _load_balance(self, request):
                         # Close the stream response once all chunks are sent
                         await stream_response.write_eof()
                         return stream_response
-
-                elif request.method == 'POST':
-                    d = await request.read()
-                    import json
-
-                    async with session.post(
-                        url=target_url, json=json.loads(d.decode())
-                    ) as response:
-                        content = await response.read()
-                        return web.Response(
-                            body=content,
-                            status=response.status,
-                            content_type=response.content_type,
-                        )
+                    content = await response.read()
+                    return web.Response(
+                        body=content,
+                        status=response.status,
+                        content_type=response.content_type,
```

Comment:
should the header be added here as well?

Reply:
yes this would be a good idea